### PR TITLE
AnonCreds 2024 Q3 Quarterly Report

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+v2. Some well-known cryptographers have [weighed in on that approach] and
+[presented a paper] arguing that the "batch issue" approach is ineffective. As
+well, they have proposed new scheme is being proposed that implements ZKPs using
+EDDSA hardware-based keys.  The folks involved in AnonCreds are part of that
+discussion, and will see how that might impact the direction of the AnonCreds
+project.
+
+[weighed in on that approach]: https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/issues/200
+[presented a paper]: https://github.com/user-attachments/files/15904122/cryptographers-feedback.pdf
+[EUDI Wallets]: [](https://digital-strategy.ec.europa.eu/en/policies/eudi-wallet-implementation)
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md]:
```suggestion
[EUDI Wallets]: (https://digital-strategy.ec.europa.eu/en/policies/eudi-wallet-implementation)
```

File: project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+[Hyperledger AnonCreds Wiki Home Page]: https://wiki.hyperledger.org/display/ANONCREDS/
+
+## Project Health
+
+Contributions remain slow on the AnonCreds project itself this quarter, but
+there has been a significant uptake in conversations around the use of ZKP-based
+verifiable credentials in the AnonCreds and other VC communities. The EU has a
+requirement to use "unlinkable" (non-correlatable) verifiable credentials with
+the [EUDI Wallets]. To now, the plan to meet that requirement has not been to
+use ZKPs, but instead to issue a many instances of each credential and have the
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md]:
```suggestion
use ZKPs, but instead to issue many instances of each credential and have the
```

File: project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+holder use each credential just once, or at least, each credential with just one
+verifier. Each instance is slightly different, and as a result, each verifier
+gets an unlinkable credential -- although the verifiers can correlate the
+credentials through the issuer. The reasons for the "no to ZKP" has been an
+additional requirement that each credential be bound to a hardware key held in
+the holders mobile device -- which is not possible with ZKP schemes such as CL
+Signatures used in AnonCreds v1 or BBS Signatures planned for use in AnonCreds
+v2. Some well-known cryptographers have [weighed in on that approach] and
+[presented a paper] arguing that the "batch issue" approach is ineffective. As
+well, they have proposed new scheme is being proposed that implements ZKPs using
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md]:
```suggestion
well, they have proposed a new scheme that implements ZKPs using
```

File: project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+project -- [audited cryptographic libraries].
+
+[Agora]: https://labs.hyperledger.org/labs/agora.html
+[audited cryptographic libraries]: https://github.com/hyperledger-labs?q=agora
+
+[AnonCreds Rust]: https://github.com/hyperledger/anoncreds-rs
+
+## Current Plans
+
+Continued evolution of the [AnonCreds Rust] library. Contributing the
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md]:
```suggestion
Continued evolution of the [AnonCreds Rust] library. Contributing to the
```

**Review by tkuhrt — CHANGES_REQUESTED** [review]
Requesting some minor changes. Looks good otherwise.

**Review by petermetz — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by arsulegai — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+verifier. Each instance is slightly different, and as a result, each verifier
+gets an unlinkable credential -- although the verifiers can correlate the
+credentials through the issuer. The reasons for the "no to ZKP" has been an
+additional requirement that each credential be bound to a hardware key held in
+the holders mobile device -- which is not possible with ZKP schemes such as CL
+Signatures used in AnonCreds v1 or BBS Signatures planned for use in AnonCreds
+v2. Some well-known cryptographers have [weighed in on that approach] and
+[presented a paper] arguing that the "batch issue" approach is ineffective. As
+well, they have proposed new scheme is being proposed that implements ZKPs using
+EDDSA hardware-based keys.  The folks involved in AnonCreds are part of that
```
Comment (JoLuPuma) [inline project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md]: @swcurran 
Is this about the Dfinity (Google) proposal? did they mainly talk about ECDSA (and not EDDSA)?

Comment (swcurran) [reply project-reports/2024/2024-Q3-Hyperledger-AnonCreds.md]: Good catch @JoLuPuma — thanks.  Yes — this is the Google et. al. proposal.

**Review by jimthematrix — APPROVED** [review]

**Review by tkuhrt — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Pull request merged**